src/code/z_actor_dlftbls.c

```diff
@@ -100,8 +100,8 @@ void ActorOverlayTable_FaultPrint(void* arg0, void* arg1) {
     u32 overlaySize;
     uintptr_t ramStart;
     uintptr_t ramEnd;
-    u32 offset;
```

Comment:
what about marking this UNUSED_NDEBUG instead to minimize the amount of preprocessor noise?

Reply:
I'd have to introduce a new define for UNUSED predicated on PLATFORM_N64 instead of DEBUG_FEATURES to do this, which I don't mind doing if that sounds better to you

Reply:
oh right, if this is the only place it would be used, then never mind